gammapy/maps/tests/test_core.py

```diff
@@ -990,3 +990,31 @@ def test_data_shape_broadcast():
 
     with pytest.raises(ValueError):
         map2.data = np.ones((1, 1, 2))
+
+
+def test_make_mask_geom():
+    energy = MapAxis.from_energy_bounds("1 TeV", "10 TeV", nbin=5)
+    phase = MapAxis.from_bounds(0, 1, nbin=6, name="phase")
+    freq = MapAxis.from_bounds(1, 10, nbin=4, unit=u.Hz, name="freq")
+
+    geom = WcsGeom.create(10, axes=[energy, phase, freq])
```

Comment:
I think you should test other geoms. A minima HpxGeom as it has a different shape that WCSgeom and RegionGeom

Reply:
Done